extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -226,28 +208,12 @@ public void run() {
         }
     }
 
-    /**
-     * For tests only. Blocks the current thread until all messages are written
-     * Just does a busy wait.
-     * <p/>
-     * DO NOT USE THIS OUTSIDE OF A TEST
```

Comment:
seems not used at all.

Reply:
but duplicated: 

https://github.com/search?q=org%3Aquarkusio%20awaitWrittenForTest&type=code